test/Container/ServiceManager/AutowireFactoryTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Di\Container\ServiceManager;
+
+use PHPUnit\Framework\TestCase;
+use Zend\Di\Container\AutowireFactory as GenericAutowireFactory;
+use Zend\Di\Container\ServiceManager\AutowireFactory;
+use Interop\Container\ContainerInterface;
+
+/**
+ * AutowireFactory test case.
+ *
+ * @coversDefaultClass Zend\Di\Container\ServiceManager\AutowireFactory
+ */
+class AutowireFactoryTest extends TestCase
+{
+    /**
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    private function createGenericFactoryMock()
+    {
+        return $this->getMockBuilder(GenericAutowireFactory::class)
+                    ->setMethodsExcept()
+                    ->getMock();
```

Comment:
Reflow these two lines at a single extra level of indentation (instead of the three or so you have here).